po/cs.po

```diff
@@ -2039,7 +2039,7 @@ msgstr "O jeden krok workflow vpřed (Ctrl+Enter)"
 
 #: ../virtaal/views/widgets/aboutdialog.py:34
 msgid "Copyright © 2007-2010 Zuza Software Foundation"
-msgstr "Copyright © 2007-2011 Zuza Software Foundation"
+msgstr "Copyright © 2007-2010 Zuza Software Foundation"
```

Comment:
I wouldn't worry about this number as its part of a date range.  Likely the source should say 2007-2016.  So its OK to land, just not critical and not more correct.